docs/docs/administration/08-advanced/01-external-authentication.md

```
@@ -126,15 +126,82 @@ To disable the shibboleth session check you can set the following `.env` variabl
+#### Key Requirements
+
+- ##### Client Registration
+
+    Manual registration with your OpenID Connect Provider is required. Dynamic registration is **not supported**.
+
+- ##### Client Authentication
+
+    The application authenticates with the token endpoint using the `client_secret_basic` method.
+
+- ##### Supported Signing Algorithms
+    - RS256, RS384, RS512
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)